mifospay/src/main/res/values/strings.xml

```diff
@@ -175,8 +175,9 @@
     <string name="pin_code">Pin Code</string>
     <string name="state">State</string>
     <string name="no_transactions_found">No transactions found</string>
-    <string name="change_profile_picture">Change Profile Picture</string>
-    <string name="remove_profile_picture">Remove Profile Picture</string>
+    <string name="click_profile_picture">Click profile picture</string>
+    <string name="change_profile_picture">Pick profile picture from device/gallery</string>
```

Comment:
Let's change the message to `Pick profile picture from device`. 
Earlier I was trying to say that we can use any one of them (device or gallery) for the text that will be displayed to the user

Reply:
Thanks, I've made the changes.